pkg/pillar/cmd/zedrouter/appcontainer.go

```diff
@@ -42,8 +53,9 @@ func appCheckStatsCollect(ctx *zedrouterContext, config *types.AppNetworkConfig,
 }
 
 // goroutine for App container stats collection
-func appStatsCollect(ctx *zedrouterContext) {
-	log.Infof("appStatsCollect: containerStats, started")
+func appStatsAndLogCollect(ctx *zedrouterContext) {
+	log.Infof("appStatsAndLogCollect: containerStats, started")
+	lastLogTime := make(map[string]string)
```

Comment:
Would be good to document 		// remember the last entry time by a container
and that the map is from containername string to a timestamp string.

Reply:
will do.